src/customize-pop/gnome-tweaks-extensions/user-themes.md

```diff
@@ -0,0 +1,94 @@
+
+>**Note**: Applying custom GNOME theming options may cause system instability or glitchy behavior. Additionally, custom themes are not subject to testing by the System76 Quality Assurance team or application developers.
+
+## Enabling User Themes
+
+1. Launch the GNOME Extension Manager.
+
+    ![Launch GNOME Extension Manager](/images/user-themes/launch-ext-manager.png)
+
+2. Click `Browse` and search for `user themes`, then click `Install`.
+
+    ![Install User Themes](/images/user-themes/install-user-themes.png)
+
+3. Click `Installed` and ensure the toggle switch for `User Themes` is enabled.
+
+    ![Verify User Themes is Enabled](/images/user-themes/verify-enabled.png)
+
+## Adding a Custom User Theme
+
+> **Note**: Themes in Pop 22.04 must support GTK 4. Additionally, it may be necessary to disable [Pop!\_OS components](gnome-tweaks-extensions.md#built-in-extensions) from GNOME Extensions.
+
+### Downloading a Custom Theme
+
+Custom GNOME user themes can be downloaded from [opendesktop.org](https://www.opendesktop.org/s/Gnome/browse/).
+
+![GNOME Themes on OpenDesktop.org](/images/user-themes/open-desktop-themes.png)
+
+Download files for the theme are usually listed under `Files` section on the theme's page.
+
+### Installing a Custom Theme
+
+Follow the instructions on the theme's page to add it to your system, these are typically listed in the `Product` section on the theme's page. If no instructions are listed, click the link provided next to `Source` and view the theme's README.
+
+![Theme Installation Instructions](/images/user-themes/installation-instructions.png)
+
+Installation methods will vary, but the process will usually involve extracting a tar.gz file to either */usr/share/themes* or *~/.themes*. In the example below, a theme called `example-theme.tar.xz` is extracted to *usr/share/themes*.
+
+```bash
+sudo tar -xf example-theme.tar.xz -C /usr/share/themes
+```
+
+### Applying a Custom Theme
+
+Launch GNOME Tweaks, then select the `Appearance Tab` and choose a theme from the `Shell` dropdown menu.
+
+![Select Theme from Appearance Tab](/images/user-themes/select-theme.png)
+
+You may need to restart GNOME Tweaks if it was open when extracting the theme to your local directory. Additionally, you may also need to restart the GNOME Shell by pressing `Alt` + `F2`, then typing `r` and hitting `Enter`.
+
+## Resetting the User Theme to Default
+
+You can reset user themes to the Pop!\_OS defaults using GNOME Tweaks and Extensions, or by entering Terminal commands.
+
+### Using GNOME Tweaks and Extensions
+
+Launch GNOME Tweaks, then select the `Appearance Tab` and choose a theme from the `Shell` dropdown menu.
+
+![Select Theme from Appearance Tab](/images/user-themes/reset-theme.png)
+
+Launch GNOME Extensions and re-enable any built-in extensions you may have disabled when applying a custom theme.
```

Comment:
May also want to mention that the user should disable any additional extensions they have installed if they want to get back to a default COSMIC experience.